src/dymaptic.GeoBlazor.Core/Model/AuthenticationManager.cs

```diff
@@ -56,18 +59,51 @@ public string? PortalUrl
         {
             if (string.IsNullOrWhiteSpace(_portalUrl))
             {
-                _portalUrl = _configuration["ArcGISPortalUrl"];
+                var fromConfig = _configuration["ArcGISPortalUrl"];
+                _portalUrl = NormalizePortalUrl(fromConfig);
             }
-
             return _portalUrl;
         }
         set
         {
-            if (!string.IsNullOrWhiteSpace(value))
+            _portalUrl = NormalizePortalUrl(value);
+        }
+    }
+
+    private static string? NormalizePortalUrl(string? input)
+    {
+        if (string.IsNullOrWhiteSpace(input)) return "https://www.arcgis.com";
```

Comment:
This can't be right. We should not be adding `/portal` sometimes and not other times. And in the typescript file, we still have some methods with `/portal` and some without.

Let's talk tomorrow through this and look at it together. If we have a source of documentation for this we should be adding links as well.

Reply:
After our discussion this morning, and removing the extra `/portal` in the TypeScript file, I believe this method is correct. However, please move it to the bottom of the file, below the public methods.